profiles/views.py

```diff
@@ -220,3 +221,41 @@ def form_valid(self, form):
 
 def about(request, template_name='about.html'):
     return render(request, template_name,)
+
+
+class BulkUser(LoginRequiredMixin, View):
+    '''
+    Custom class to create bulk student users.
+    '''
+
+    def get(self, request):
+        username = request.user
+
+        if username.is_superuser:
+            return render(request, "bulkuserform.html")
+        else:
+            return render(request, "404.html")
+
+    def post(self, request):
+        username = request.user
+
+        if username.is_superuser:
+            format = request.POST.get("format")
```

Comment:
you should avoid to use `format` as the variable name since its a keyword :stuck_out_tongue: